src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -730,6 +730,146 @@ export class BaseEditCourseForm extends React.Component {
                   disabled={disabled}
                 />
                 )}
+                <Field
```

Comment:
These fields will not be displayed for every course. Only for Bootcamps and Executive Education courses.